pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -161,7 +161,7 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 		var ok bool
 		newRc, ok = obj.(*api.ReplicationController)
 		if !ok {
-			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController", filename)
+			return cmdutil.UsageError(cmd, "%s does not specify a valid ReplicationController (%#v)", filename, obj)
```

Comment:
That serialization tends to look pretty ugly and it's not what a user is accustomed to seeing.  Perhaps attempting  a json encoding using `latest`?